api/CHANGELOG.md

```diff
@@ -43,3 +43,11 @@ other smart filter endpoints.
 
 Fixed 
 Impact table - fix filters when retrieving 'group by' entities for table skeleton
+
+## 2022-06-30
+
+### Fixed
```

Comment:
Very minimal, I believe this is a not actually a bug fix but a change of requirements. The previous implementation met the previous specs, so maybe it would be more accurate to reflect that as well, up to you